src/NotificationBundle/Form/Type/TransportSettingType.php

```
@@ -45,19 +46,41 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
 
         $builder = new DynamicFormBuilder($builder);
 
-        $builder
-            ->add('name')
-            ->add('transport', ChoiceType::class, [
+        $builder->add('name');
+
+        // If transport is pre-filled (from URL), use hidden field. Otherwise show dropdown.
+        $data = $builder->getData();
+        $isNewWithPreselectedTransport = false;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$isNewWithPreselectedTransport'. (undefined)

(UnusedLocalVariable)